ansible_runner/utils.py

```diff
@@ -95,8 +97,21 @@ def stream_dir(directory):
                     archive.write(os.path.join(dirpath, fname), arcname=os.path.join(relpath, fname))
         archive.close()
 
-    payload = buf.getvalue()
-    return b'\n'.join((json.dumps({'zipfile': len(payload)}).encode('utf-8'), payload))
+    zip_size = Path(zip_file).stat().st_size
```

Comment:
I definitely had to test that this gives the same answer as `len(f.read())`, because that was not at all obvious to me.